tests/utils/test_dag_processing.py

```diff
@@ -50,7 +51,7 @@
 from tests.test_utils.config import conf_vars
 from tests.test_utils.db import clear_db_dags, clear_db_runs, clear_db_serialized_dags
 
-TEST_DAG_FOLDER = os.path.join(os.path.dirname(os.path.realpath(__file__)), os.pardir, 'dags')
+TEST_DAG_FOLDER = pathlib.Path(__file__).parent.parent / 'dags'
```

Comment:
Should we just do since everywhere we make it `str`:

```python
TEST_DAG_FOLDER = str(pathlib.Path(__file__).parent.parent / 'dags')
```

Reply:
If we make this a str here we can't use `TEST_DAG_FOLDER / "some_dag"` that I do elsewhere.

Reply:
A bit of a bigger change, but probably worth it.